src/Core/MySQL/MySQLGtid.cpp

```diff
@@ -0,0 +1,178 @@
+#include "MySQLGtid.h"
+
+#include <boost/algorithm/string.hpp>
+
+namespace DB
+{
+namespace ErrorCodes
+{
+    extern const int UNKNOWN_EXCEPTION;
+}
+
```

Comment:
Code for parsing and updating GTID sets looks suboptimal (but it's ok for now and can be optimized later if necessary)